src/types/create_item.rs

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use serde::Deserialize;
+use xml_struct::XmlSerialize;
+
+use crate::{
+    types::sealed::EnvelopeBodyContents, BaseFolderId, Items, Operation, OperationResponse,
+    ResponseClass, ResponseCode, MESSAGES_NS_URI,
+};
+
+/// Describes how an item is handled once it has been created, if it's a message
+/// item.
```

Comment:
suggestion: This is a little clearer about what actually happens, I think.

```suggestion
/// The action an Exchange server will take upon creating a `Message` item.
```